packages/amplify/amplify_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 1.0.0-next.0 (2022-08-02)
+
+Initial developer preview release.
```

Comment:
```suggestion
Developer preview release for web and desktop support.
```

Reply:
We probably want to be specific here (and other changelogs), since we already had our 'initial' developer preview back when the library first launched.

Reply:
If I were to put that in all CHANGELOGs, it would be misleading since only Auth has Web/Desktop support. That's why I changed the other copy to elaborate more on the incremental release.

Reply:
I think this could make sense:
> Initial developer preview release for all platforms.

It is the initial release for the all platforms, so I think that part is worth highlighting. The phrasing is just pretty challenging given the platform split between the packages.